pkg/accounting/common.go

```diff
@@ -18,4 +18,6 @@ const (
 	LastAtRestTally = "LastAtRestTally"
 	// LastBandwidthTally represents the accounting timestamp for the bandwidth allocation query
 	LastBandwidthTally = "LastBandwidthTally"
+	// LastAtReLastRollupstTally represents the accounting timestamp for rollup calculations
```

Comment:
is this name right "LastAtReLasRollupstTally"? Looks like something got dragged and dropped accidentally